cli/dts/lib.deno.unstable.d.ts

```diff
@@ -390,17 +390,17 @@ declare namespace Deno {
     ? void
     : T extends StaticNativeBigIntType ? bigint
     : T extends StaticNativeNumberType ? number
-    : T extends "pointer" ? UnsafePointer
+    : T extends "pointer" ? bigint
     : never;
 
   type StaticForeignFunctionParameter<T> = T extends "void" ? void
     : T extends StaticNativeNumberType | StaticNativeBigIntType
       ? number | bigint
-    : T extends "pointer" ? UnsafePointer | TypedArray | null
+    : T extends "pointer" ? bigint | TypedArray | null
     : T extends NativeCallbackType<
       infer U extends readonly NativeType[],
       infer V extends NativeParameterType
-    > ? UnsafeCallback<U, V> | UnsafePointer | null
+    > ? UnsafeCallback<U, V> | bigint | null
```

Comment:
With the removal of the wrapping, passing `UnsafeCallback<U, V>` as a parameter will not work.